src/screens/Library/components/GameCard/index.css

```diff
@@ -55,21 +52,31 @@
   height: 20px;
 }
 
-.gameCard > a > .gameImg {
-  width: 8vw;
-  height: 10vw;
-  max-width: 200px;
-  max-height: 266px;
-  min-width: 130px;
-  min-height: 173px;
+.gameCard > a {
```

Comment:
There are many changes in this file, some are new styles, some are just me moving things around so similar elements are together, and are refactors like moving width, min-width and max-width to a widh: clamp(...) property and things like that